lib/OpenQA/Setup.pm

```diff
@@ -222,10 +223,11 @@ sub read_config {
               exists $mode_defaults{$app->mode}{$section}->{$k}
               ? $mode_defaults{$app->mode}{$section}->{$k}
               : $defaults{$section}->{$k};
-            $app->config->{$section}->{$k} = $v if defined $v;
+            $app->config->{$section}->{$k} = trim $v if defined $v;
         }
     }
-    $app->config->{global}->{recognized_referers} = [split(/ /, $app->config->{global}->{recognized_referers})];
+
```

Comment:
could you please delete this blank line? I think it sneaked in by mistake

Reply:
ok, I will delete it